src/main/java/com/github/wovnio/wovnjava/Interceptor.java

```diff
@@ -16,7 +16,7 @@
     }
 
     String translate(String body) {
-        String lang = headers.getRequestLang();
+        String lang = headers.requestLang;
```

Comment:
I am not a huge fan of having `requestLang` public. It breaks uniform access, now we use either public attributes or getters. I prefer using getters for consistency.

Reply:
Headers currently has almost no getters, and fields are simply accessed directly. Consistency points to not using getters, if anything. That said, the old fields that are accessed directly will likely be removed soon.

I think I liked accessing the fields as if Headers is a struct, essentially. But I'm fine with using getters as well.